tests/utils_tests/test_text.py

```diff
@@ -225,3 +226,16 @@ def test_compress_sequence(self):
         out = text.compress_sequence(seq)
         compressed_length = len(b''.join(out))
         self.assertTrue(compressed_length < actual_length)
+
+    def test_format_lazy(self):
+        self.assertEqual('django/test', six.text_type(format_lazy('{}/{}', 'django', 'test')))
```

Comment:
Are all the `six.text_type` needed? Tests seem to be passing with at least the first one removed.

Shouldn't there be tests with lazy versions of args, kwargs? It looks like these tests are mainly testing `str.format()`.